README.md

```diff
@@ -39,11 +39,21 @@ Compared with other graph database solutions, **Nebula Graph** has the following
 * SQL-like query language
 * Role-based access control for higher level security
 
+## About this Repo
+
+Please note that this repo is only for Nebula Graph `v1.*.*`.
+
+From `v2.*.*`, which was already GA in 2021 April, Nebula Graph's core code bases are [nebula-graph](https://github.com/vesoft-inc/nebula-graph), [nebula-common](https://github.com/vesoft-inc/nebula-common) and [nebula-storage](https://github.com/vesoft-inc/nebula-storage), it's recommended to start from the repo: [nebula-graph](https://github.com/vesoft-inc/nebula-graph).
+
 ## Quick start
 
-Read the [Getting started](https://docs.nebula-graph.io/manual-EN/1.overview/2.quick-start/1.get-started/) guide to quickly get going with **Nebula Graph**.
+Read the `Getting started` from [docs](https://docs.nebula-graph.io/) to quickly get going with **Nebula Graph**.
 
-Please note that you must install **Nebula Graph** by [installing source code](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code/), [rpm/deb packages](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb/) or [docker compose](https://github.com/vesoft-inc/nebula-docker-compose), before you can actually start using it. If you prefer a video tutorial, visit our [YouTube channel](https://www.youtube.com/channel/UC73V8q795eSEMxDX4Pvdwmw).
+For Nebula Graph Installation, in addition to referring to the docs to deploy from one of the three options: source code, RPM/DEB package, or Docker Compose, if you prefer a video tutorial, please check out our [YouTube channel](https://www.youtube.com/channel/UC73V8q795eSEMxDX4Pvdwmw).
```

Comment:
installation

Optimize this paragraph according to the comment for the Chinese version?

Reply:
sure, will do so.

Reply:
done